samples/subsys/portability/cmsis_rtos_v1/philosophers/src/main.c

```diff
@@ -100,19 +100,22 @@ static void print_phil_state(int id, const char *fmt, int32_t delay)
 	int prio = osThreadGetPriority(osThreadGetId());
 
 	set_phil_state_pos(id);
+	char state[80];
```

Comment:
It would be nice to make the constant "80" a macro since it is referenced multiple times (or make any references to it in terms of `ARRAY_SIZE(state)`; but, that is probably slightly longer than how long we want it to be).

Reply:
good point. I've switched to using a macro for the size, which required some minor reformatting.